pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -601,7 +601,7 @@ func validateBootOrder(field *k8sfield.Path, spec *v1.VirtualMachineInstanceSpec
 
 		matchingVolume, volumeExists := volumeNameMap[disk.Name]
 
-		if !volumeExists {
+		if !volumeExists && disk.CDRom == nil {
```

Comment:
This should be guarded with a feature gate.

In general, we should try our best to ensure that this PR's changes will have absolutely no affect unless the relevant feature gate is enabled.